src/widgets.ts

```diff
@@ -13,6 +13,8 @@ type BaseOptions = {
   container: string | Element
   iframeTitle?: string
   style?: Partial<CSSStyleDeclaration>
+  url?: string
+  proxy?: string
```

Comment:
We don't need these types here, they come from `UrlLoadingProps` if I remember correctly and that type is already applied. If removing these types here causes a compiler error, then the fix will be something else.